.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  push:
```

Comment:
Use 4 space indentation?

Reply:
I just copied this from github actions template, I think 2 space is conventional here